lib/esConnector.js

```diff
@@ -583,7 +583,7 @@ ESConnector.prototype.buildDeepNestedQueries = function (root, idName, where, bo
 			else {
 				var orObject = {bool: {should: []}};
 				orPath = orObject.bool.should;
-				path.push(orPath);
+				path.push(orObject);
```

Comment:
@aquid - questions:
1. why didn't the testcase catch this?
2. is this a valid fix?